### PR TITLE
Add links to Progress wiki and contact section

### PR DESCRIPTION
**Pull request by foursixnine**
Fix a small formatting error and add new links to the readme:

Also being enabled in openQA project: https://github.com/os-autoinst/openQA/pull/5030 

# Preview
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,39 @@
...
 [documentation for UI Test Automation Framework](ui-framework-documentation.md)
 
 ## How to contribute
 Please, refer to [Contributing Guide](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md).
 
+## How to get in touch?
+
+Reach us via http://open.qa/contact, we are also on twitter as https://twitter.com/openQAhq
+
+You can also read about some of the teams at https://progress.opensuse.org/projects/qa/wiki
```
Comment (Martchus) [inline README.md:35]:
```suggestion
Reach us via http://open.qa/contact, we are also on Twitter as https://twitter.com/openQAhq.

You can also read about some of the teams at https://progress.opensuse.org/projects/qa/wiki.
```

File: README.md [hunk 2]
```diff
@@ -1,30 +1,38 @@
...
-In case of adding new test for Installation, please use approach
-described in the
+In case of adding new test for Installation, please use approach described in the
 [documentation for UI Test Automation Framework](ui-framework-documentation.md)
 
 ## How to contribute
 Please, refer to [Contributing Guide](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md).
 
+## How to get in touch?
+Reach us via http://open.qa/contact, we are also on twitter as https://twitter.com/openQAhq
```
Comment (fgerling) [inline README.md:32]: I guess the links will be auto-linked with goldmark/githubs markdown, and such don't need to be explicitly formatted as links.

**Review by fgerling — APPROVED** [review]
Looks good to me. I followed the new links, everything worked.

File: README.md [hunk 3]
```diff
@@ -1,30 +1,38 @@
...
 
 For using new mechanism to schedule modules, check [declarative schedule docs](declarative-schedule-doc.md)
 
-In case of adding new test for Installation, please use approach
-described in the
+In case of adding new test for Installation, please use approach described in the
 [documentation for UI Test Automation Framework](ui-framework-documentation.md)
 
 ## How to contribute
 Please, refer to [Contributing Guide](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md).
```
Comment (rfan1) [inline README.md:29]: A very small comment:
remove the '.' at the end

File: README.md [hunk 4]
```diff
@@ -1,30 +1,38 @@
...
 
+## How to get in touch?
```
Comment (rfan1) [inline README.md:31]: A very small comment:

remove the '?' at the end

**Review by rfan1 — APPROVED** [review]
LGTM

File: README.md [hunk 5]
```diff
@@ -1,29 +1,36 @@
...
+
 os-autoinst-distri-opensuse is repo which contains tests, which are executed
 by openQA for openSUSE and SLE distributions.
 
 Needles for openSUSE distributions are located in [os-autoinst-needles-opensuse](https://github.com/os-autoinst/os-autoinst-needles-opensuse)
 
 Some documentation from the test library is published at https://os-autoinst.github.io/os-autoinst-distri-opensuse/, however this is still WIP
 
-For more details see http://os-autoinst.github.io/openQA/
+For more details see http://open.qa/
```
Comment (grisu48) [inline README.md:18]: Please let's use https

```suggestion
For more details see https://open.qa/
```

**Review by Martchus — APPROVED** [review]

**Pull request merged**